supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -8,10 +8,21 @@ Follow the _IBM Style_ guide advice of _day Month year_ for date formats, for ex
 
 When the format _day Month year_ causes a presentation or clarity issue, use _Month day, year_ (for example, October 3, 2019) instead.
 
+[[man-pages]]
+== Man page references
+
+When referencing a man page, use the following format: `<name_of_utility>(<section_number>)` man page.
```

Comment:
```suggestion
When referencing a man page, use the following format: _`<name_of_utility>(<section_number>)` man page_.
```
Italicize the template text to make it clear what is part of the template and what isn't. This formatting is used for date formats.

Reply:
Thanks, updated!

Reply:
Actually one sec, that messed up the backtick formatting.

Reply:
Okay, updated